### PR TITLE
Fixing up broken tests due to the compositor refactor

### PR DESCRIPTION
**Pull request by jaredhms**
This change is [Reviewable](https://reviewable.io/reviews/microsoft/winobjc/1360)

### CODE REVIEW

**Comment by jaredhms** [comment]
@yiyang-msft is added to the review. #Closed

**Comment by jaredhms** [comment]
@oliversa-msft is added to the review. #Closed

**Comment by jaredhms** [comment]
@davelamb is added to the review. #Closed

**Comment by yiyang-msft** [comment]
:shipit:

**Comment by oliversa-msft** [comment]
:shipit:

File: tests/functionaltests/Tests/UIViewTests.mm [hunk 1]
```diff
@@ -27,12 +28,146 @@
...
+    WXFrameworkElement* backingElement = [view xamlElement];
+    ASSERT_TRUE(backingElement);
+    ASSERT_TRUE([backingElement isKindOfClass:[WXFrameworkElement class]]);
+    ASSERT_TRUE([backingElement.name isEqualToString:@"UIView"]);
+}
+
+class UIViewTestHelper {
+public:
+    UIViewTestHelper() {
+        _rootView = [[UIView alloc] init];
```
Comment (oliversa-msft) [inline tests/functionaltests/Tests/UIViewTests.mm]: >[[UIView alloc] init] [](start = 20, length = 21)

Should these be also autoreleased?

**Pull request merged**